src/test/denialofservice_tests.cpp

```diff
@@ -288,7 +287,7 @@ BOOST_AUTO_TEST_CASE(DoS_banscore)
     dummyNode1.fSuccessfullyConnected = true;
     {
         LOCK(cs_main);
-        Misbehaving(dummyNode1.GetId(), 100);
+        Misbehaving(dummyNode1.GetId(), DISCOURAGEMENT_THRESHOLD - 11);
```

Comment:
Why the change from `100` to `89` here? :)

Reply:
See the two removed `-banscore` lines of this test, particularly the first one: `gArgs.ForceSetArg("-banscore", "111"); // because 11 is my favorite number`.